waspc/data/Generator/templates/server/src/auth/providers/local/login.ts

```diff
@@ -1,35 +1,25 @@
 {{={= =}=}}
-import Prisma from '@prisma/client'
-import SecurePassword from 'secure-password'
-
-import { sign, verifyPassword } from '../../../core/auth.js'
+import { verifyPassword } from '../../../core/auth.js'
 import { handleRejection } from '../../../utils.js'
 
-const prisma = new Prisma.PrismaClient()
+import { findUserBy, createAuthToken } from '../../utils.js'
 
 export default handleRejection(async (req, res) => {
   const args = req.body || {}
 
-  // Try to fetch user with the given username.
-  const user = await prisma.{= userEntityLower =}.findUnique({ where: { username: args.username } })
+  const user = await findUserBy<'email'>({ email: args.email })
```

Comment:
This appears to break `usernameAndPassword` auth.

Reply:
Should be fixed now 👍